### PR TITLE
[docs-only] ADR-0026 Application based user settings

### PR DESCRIPTION
**Pull request by butonic**
something for web!

### CODE REVIEW

**Comment by butonic** [comment]
cc @DeepDiver1975 @dragotin this PR is only about the libre graph api. But we also need to decide where to put the data. IMO that should be a dedicated space per app in the systems storage.

**Review by butonic — COMMENTED** [review]
mark as draft

File: docs/ocis/adr/0026-application-based-user-settings.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+geekdocFilePath: 0026-application-based-user-settings.md
+---
+
+* Status: draft
+* Deciders: @butonic, @fschade, @kulmann
+* Date: 2024-02-09
+
+## Context and Problem Statement
+
+To share user settings across devices applications want to store user specific settings on the server. The ePUB app wants to remeber which page the user is on. The iOS app wants to rember search queries. The Caldav app needs a space to store data.
```
Comment (kulmann) [inline docs/ocis/adr/0026-application-based-user-settings.md:16]:
```suggestion
To share user settings across devices applications want to store user specific settings on the server. The ePUB app wants to remember which page the user is on. The iOS app wants to rember search queries. The Caldav app needs a space to store data.
```

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**